core/storage/user/gae_models.py

```diff
@@ -1100,6 +1168,11 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.DELETE
 
+    @staticmethod
+    def get_export_method():
+        """Model is exported as a single shared instance."""
```

Comment:
Implementation does not match docstring :(

Reply:
done